config_example.py

```diff
@@ -1,8 +1,9 @@
 ACCESS_TOKEN = "123456:ABC-DEF1234ghIkl-zyx57W2v1u123ew11"
+DEV_ID = 2011545269
 GROUP_CHAT_ID = -1001853277651
```

Comment:
btw it's probably better to have GROUP_CHAT_ID as an array, so that we can have multiple groups that have full support of this bot for its members if needed with no code change. But i wouldn't suggest to change it in this pull request. better to change it sometime in future

Reply:
maybe I can try that and make a separate commit before merge this.

Reply:
but checking if a member was in a group probably send a request, if there are multiple groups, it will send multiple requests each time any user send a message, it does not sounds good.
also I don't think we will have the needs of this, even if we have multiple groups, we probably don't need a music bot to serve both group which was in the same topic (for music sharing). maybe just leave it. at least we don't need that for now.

Reply:
I think about this again, it's a good idea. I'll need this for checking if the user is member of a group or channel, this will be useful, and with a for loop, `return True` when it founds that he was in a group/channel, it probably won't cost all loop to check if he was in one of those. I'll add commit for this before merge this PR.

Reply:
But need to come up with another name than `GROUP_CHAT_ID` because it could also be a channel. `PROMOTION_CHATS`? (since both telegram channels, groups, and private chat are in "Chat" object, I think it's ok to use the word CHATS, and for users who are not in the group, it's basically a PROMOTION)